Client/WriteApiAsync.cs

```diff
@@ -294,7 +293,7 @@ await WriteData(options.OrganizationId, options.Bucket, grouped.Key, groupedPoin
         public Task WritePointsAsync(PointData[] points, string bucket = null, string org = null,
             CancellationToken cancellationToken = default)
         {
-            return WritePointsAsync(points.ToList(), bucket, org, cancellationToken);
+            return WritePointsAsync(points, bucket, org, cancellationToken);
```

Comment:
This causes infinity recursive, please change to:
```suggestion
            return WritePointsAsync((IEnumerable<PointData>)points, bucket, org, cancellationToken);
```